astroquery/vizier/core.py

```diff
@@ -62,6 +62,16 @@ def __init__(self, columns=["*"], column_filters={}, catalog=None,
             column descriptions defined on the Vizier web pages.
             See http://vizier.u-strasbg.fr/vizier/vizHelp/1.htx#ucd and
             http://cds.u-strasbg.fr/w/doc/UCD/
+         vizier_server : string
```

Comment:
The whitespace is misaligned here, I believe
```suggestion
        vizier_server : string
```